File: ZoneSentry/Models/RentPayment.cs
namespace ZoneSentry.Models;

public class RentPayment
{
    public int Id { get; set; }
    
    public int Payment { get; set; }
    public PaymentState PaymentState { get; set; }
    public DateTime DueDate { get; set; }
    public DateTime PeriodStart { get; set; }
    public DateTime PeriodEnd { get; set; }
    
    public RentAgreement RentAgreement { get; set; }
}

File: ZoneSentry/ClientApp/src/api/models/RentPayment.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { PaymentState } from './PaymentState';
import type { RentAgreement } from './RentAgreement';

export type RentPayment = {
    id?: number;
    payment?: number;
    paymentState?: PaymentState;
    dueDate?: string;
    periodStart?: string;
    periodEnd?: string;
    rentAgreement?: RentAgreement;
};